types/spotify-api/index.d.ts

```diff
@@ -468,10 +468,18 @@ declare namespace SpotifyApi {
      * Get user's saved shows
      * 
      * GET /v1/me/shows
-     * https://developer.spotify.com/documentation/web-api/reference/library/get-users-saved-shows/
+     * https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-users-saved-shows/
      */
     type UsersSavedShowsResponse = PagingObject<SavedShowObject>;
 
+    /**
+     * Get User's Saved Episodes
+     *
+     * GET /v1/me/episodes
+     * https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-users-saved-episodes/
```

Comment:
Doesn't work if the slash is added to the end of the anchor
```suggestion
     * https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-users-saved-episodes
```

Reply:
Good catch! Fixed it.